pkg/eve/runme.sh

```diff
@@ -9,6 +9,7 @@ OUTPUT_IMG=/tmp/output.img
 DEFAULT_LIVE_IMG_SIZE=592
 DEFAULT_INSTALLER_IMG_SIZE=592
 DEFAULT_NVIDIA_IMG_SIZE=900
+DEFAULT_KUBEVIRT_IMG_SIZE=1024
```

Comment:
@andrewd-zededa I've tested locally and is still not working... 

https://github.com/lf-edge/eve/blob/master/Makefile#L412

Size for kubevirt rootfs is 900MB, so apparently 1024 is still not enough to accommodate other partitions + installer files...